src/bindbc/icu/bindings/parseerr.d

```diff
@@ -0,0 +1,117 @@
+/*******************************************************************************
+ * parseerr.h
+ * 
+ * C API: Parse Error Information
+ * 
+ * License:
+ * COPYRIGHT AND PERMISSION NOTICE (ICU 58 and later)
+ * 
+ * Copyright © 1991-2020 Unicode, Inc. All rights reserved.
+ * Distributed under the Terms of Use in https://www.unicode.org/copyright.html.
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of the Unicode data files and any associated documentation
+ * (the "Data Files") or Unicode software and any associated documentation
+ * (the "Software") to deal in the Data Files or Software
+ * without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, and/or sell copies of
+ * the Data Files or Software, and to permit persons to whom the Data Files
+ * or Software are furnished to do so, provided that either
+ * (a) this copyright and permission notice appear with all copies
+ * of the Data Files or Software, or
+ * (b) this copyright and permission notice appear in associated
+ * Documentation.
+ * 
+ * THE DATA FILES AND SOFTWARE ARE PROVIDED "AS IS", WITHOUT WARRANTY OF
+ * ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+ * WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT OF THIRD PARTY RIGHTS.
+ * IN NO EVENT SHALL THE COPYRIGHT HOLDER OR HOLDERS INCLUDED IN THIS
+ * NOTICE BE LIABLE FOR ANY CLAIM, OR ANY SPECIAL INDIRECT OR CONSEQUENTIAL
+ * DAMAGES, OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE,
+ * DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER
+ * TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+ * PERFORMANCE OF THE DATA FILES OR SOFTWARE.
+ * 
+ * Except as contained in this notice, the name of a copyright holder
+ * shall not be used in advertising or otherwise to promote the sale,
+ * use or other dealings in these Data Files or Software without prior
+ * written authorization of the copyright holder.
```

Comment:
Please remove the license. Instead, please follow the [previous example](https://github.com/shoo/bindbc-icu/blob/098270bc2d5e2688a50e4cfcdc542c3fd9c2ad29/src/bindbc/icu/bindings/ucnv.d#L4) and use the license notation of BSL-1.0.
As described in README/LICENSE, BSL-1.0 is applied to this library.
However, the original library (ICU4C), on which this library (bindbc-icu) depends, is distributed under the ICU license.

With the license change, it is safer to remove all copyrighted comment text.

Reply:
I'm not sure if it is allowed to remove/change the license, as the headers which I used to make the translations are licensed under their license.

e.g.
> ```
>  * Except as contained in this notice, the name of a copyright holder
>  * shall not be used in advertising or otherwise to promote the sale,
>  * use or other dealings in these Data Files or Software without prior
>  * written authorization of the copyright holder.
> ```

would no longer be in the license and users might wrongly think they (or we) can advertise with "Unicode" or similar, which we can't.

Reply:
I'm not sure if it is allowed to remove/change the license, as the headers which I used to make the translations are licensed under their license.

e.g.
> ```
>  * Except as contained in this notice, the name of a copyright holder
>  * shall not be used in advertising or otherwise to promote the sale,
>  * use or other dealings in these Data Files or Software without prior
>  * written authorization of the copyright holder.
> ```

would no longer be in the license and users might wrongly think they (or we) can advertise with "Unicode" or similar, which we can't.

Reply:
You do not need to inherit the license of the original library.

This is the same as Phobos with std.net.curl, which is licensed under BSL-1.0.
You are not obligated to display the license in order to use Phobos, but if you create an application that depends on cURL, you are obligated to display the cURL license (The curl license).

Similarly, if you use bindbc-icu, you are not obligated to display the license of bindbc-icu, but you must display the license of ICU4C.

Conversely, if you apply the same license as ICU4C to the source code of bindbc-icu, applications that depend on bindbc-icu will be obligated to display two license notices, one for bindbc-icu and one for ICU4C. It is best to avoid this.

Reply:
ok changed it